docs/hsi.xml

```diff
@@ -0,0 +1,875 @@
+      </para>
+    </refsect3>
+
+    <refsect3 id="runtime-bang">
+      <title>HSI Runtime Suffix <code>!</code></title>
+      <para>
+        A runtime security issue detected.
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            UEFI <ulink url="https://wiki.ubuntu.com/UEFI/SecureBoot">
+            Secure Boot</ulink> has been turned off. <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            The kernel is <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/tainted-kernels.html">
+            tainted</ulink> due to a non-free module or critical firmware issue. <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            The kernel is not not <ulink url="https://mjg59.dreamwidth.org/50577.html">
+            locked down</ulink>. <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            Unencrypted <ulink url="https://wiki.archlinux.org/index.php/Dm-crypt/Swap_encryption">
+            swap partition</ulink>. <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            The installed fwupd is running with <ulink url="https://github.com/fwupd/fwupd/tree/master/plugins">
+            custom or modified plugins</ulink>. <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+      </itemizedlist>
+    </refsect3>
+
+    <refsect2 id="tests">
+      <title>Tests included in fwupd</title>
+      <para>
+        The set of tests is currently x86 UEFI-centric, but will be expanded
+        in the future for various ARM or RISC-V firmware protections as required.
+        Where the requirement is architecture or processor specific it has been noted.
+      </para>
+    </refsect2>
+
+    <refsect3 id="org.fwupd.hsi.Spi.Bioswe">
+      <title>BIOS Write Enable (BWE)</title>
+      <para>
+        Intel hardware provides this mechanism to protect the SPI ROM chip
+        located on the motherboard from being overwritten by the operating system.
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            For HSI-1 the ``BIOSWE`` bit must be unset. <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+      </itemizedlist>
+      <note>
+        <para>
+          See also:
+          <itemizedlist>
+            <listitem>
+              <ulink url="http://www.intel.com/content/www/us/en/chipsets/6-chipset-c200-chipset-datasheet.html">
+                Intel C200 Datasheet
+              </ulink>
+            </listitem>
+          </itemizedlist>
+        </para>
+      </note>
+    </refsect3>
+
+    <refsect3 id="org.fwupd.hsi.Spi.Ble">
+      <title>BIOS Lock Enable (BLE)</title>
+      <para>
+        If the lock bit is set then SMIs are raised when setting BIOS Write Enable.
```

Comment:
Probably need to expand SMI as this is the first use of it.

Reply:
like?

        If the lock bit is set then System Management Interrupts (SMIs) are
        raised when setting BIOS Write Enable.

Reply:
👍